mods/modchooser/mod.yaml

```diff
@@ -4,6 +4,8 @@ Metadata:
 	Author: The OpenRA Developers
 	Hidden: true
 
+RequiresMods:
```

Comment:
Of course it's possible, but the original idea was to have it as a required field.
You can always leave it blank. I don't personally think that having optional fields here will be good for modders who don't know what they're doing.